packages/shortest/CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.1.2] - 2024-12-26
+
+### Added  
+- Added support for Mailosaur email validation
+- Added email rendering feature in the browser
+- Added sleep_milliseconds tool to add delays in the test execution
+- Added more robust error handling for Mailosaur email validation
```

Comment:
Adding the features that was added to in this PR. Since we have added a new feature, I used the wording `Added`. Maybe we can have a consistent wording for this file!?